_posts/2018-03-23-cache-breaking-for-faster-web-application-updates.md

```diff
@@ -10,26 +10,28 @@ tags: []
+"`Cache-Control: no-cache`" directs the browser to always check the server for a new version of this file. The bulk of the code for our applications are built into the JS and CSS files. For these files, I added a time stamp to the request (e.g., `/dojo/dojo.js?rel=1519311093828`). This time stamp, implemented via [grunt-cache-breaker](https://github.com/shakyShane/grunt-cache-breaker), is changed each time the application is built. Browsers cache resources by unique URL. So, when the URL to the resource is changed, the browser will instantly request a new version from the server. No more wondering whether my users are getting updates!
 
-Bonus tip: The [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) header allows the browser to check to see if a resource has changed on the server before actually downloading it (thus the `304` status code).
+_**Bonus tip**_: The [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) header allows the browser to check whether a resource has changed on the server before actually downloading it (thus the `304` status code).
 
 Here's an example of new headers for the main JavaScript file for the application:
 
 ![screenshot of headers]({{ '/images/better_headers.png' | prepend: site.baseurl }}){: .flex .flex--center .outline }
 
-Once I decided on the headers that I wanted, configuring our [IIS web server](https://docs.microsoft.com/en-us/iis/configuration/system.webserver/staticcontent/clientcache) was [fairly straight-forward](https://github.com/agrc/deq-enviro/blob/c11865a477be1d5970c457636d9c738df58483e0/_src/web.config). Here's an example of a commit that implements this entire solution: [https://github.com/agrc/deq-spills/commit/c396cca4f7fc906fd5f965888c0a992b5ae0b9df](https://github.com/agrc/deq-spills/commit/c396cca4f7fc906fd5f965888c0a992b5ae0b9df).
+Once I decided on the headers that I wanted, configuring our [IIS web server](https://docs.microsoft.com/en-us/iis/configuration/system.webserver/staticcontent/clientcache) was [fairly straightforward](https://github.com/agrc/deq-enviro/blob/c11865a477be1d5970c457636d9c738df58483e0/_src/web.config). Here's an example of a commit that implements this entire solution: [https://github.com/agrc/deq-spills/commit/c396cca4f7fc906fd5f965888c0a992b5ae0b9df](https://github.com/agrc/deq-spills/commit/c396cca4f7fc906fd5f965888c0a992b5ae0b9df).
```

Comment:
Can we add more of a conclusion-type sentence at the end? Something to kind of wrap up the ideas a little more? Right now (to me) on first read I felt like the article ended a little abruptly.